README.md

```diff
@@ -1,3 +1,48 @@
 # register-country-by-country-reporting-frontend
 
-This is a placeholder README.md for a new repository
\ No newline at end of file
+## Info
+
+This service allows a user(agent, client or organisation) to register to CBCR services.
+
+This service has a corresponding back-end service, namely register-country-by-country-reporting can be viewed [here](https://github.com/hmrc/register-country-by-country-reporting) which integrates with HOD, i.e. DES/ETMP.
+
+### Dependencies
+
+| Service          | Link                                                                   |
+|------------------|------------------------------------------------------------------------| 
+| Address Lookup   | https://github.com/hmrc/address-lookup                                 |
+| Email            | https://github.com/hmrc/email                                          |
+| Auth             | https://github.com/hmrc/auth                                           |
+| Tax Enrolments   | https://github.com/hmrc/tax-enrolments                                 |
+
+### Origin
+
+https://github.com/hmrc/register-country-by-country-reporting-frontend 
+
+### Endpoints used
+
+| Service              | HTTP Method | Route                                                                         | Purpose                                                          |
+|----------------------|-------------|-------------------------------------------------------------------------------|------------------------------------------------------------------|
+| Tax Enrolments       | POST        | /tax-enrolments/service/:serviceName/enrolment                                | Enrols a user synchronously for a given service name             
+| Registration         | POST        | /register-country-by-country-reporting/registration/noId                      | Enables user to register witout id                                                                        
```

Comment:
The service being used is Registration backend not registration "register-country-by-country-reporting"

Reply:
Mentioned service as backend service , little unsure if this is what you were asking.

Reply:
![image](https://user-images.githubusercontent.com/33547722/214248183-5cd00fac-0562-4135-823c-53a4575d6e26.png)
All of these should have "Register-country-by-country-reporting" under the service section of the table

Reply:
Fixed.